dags/common/sftp_service.py

```diff
@@ -1,6 +1,9 @@
+import os
+import traceback
```

Comment:
Do we need traceback?

Reply:
yes we do: https://github.com/cern-sis/workflows/blob/78c87c46b4e79fd6aa6e3d57f329e27659818091/dags/common/sftp_service.py#L50